### PR TITLE
Added Heatmap Release Notes

### PR DESCRIPTION
**Pull request by JainM6**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [X ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Review by dwojtowiczSumo — APPROVED** [review]
LGTM

File: blog-service/2023-05-25-metrics.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+We've enhanced the Heatmap functionality with the addition of custom dimensions on the Y-Axis. This feature enables you to gain deeper insights into your data by visualizing the frequency of occurrences for specific categories within a selected time range.
+
+Using the **Dimensional Heatmap**, you can now not only see the number of errors that occurred within a chosen time range (as with the regular Heatmap) but also effortlessly identify the specific categories, such as cluster/cpu/collector/sourceCategory, where these errors occurred most frequently.
+
+### Configure Dimension Heatmap
+
+To enable the Dimensional Heatmap with Categories on the Y-Axis, follow these steps:
+1. Run a metrics query or logs query with the **transpose** operator.
+<img src={useBaseUrl('img/metrics/custom-heat-map.png')} alt="icon" width="540"/>
+2. Change the Y-Axis Source to **Dimension**
```
Comment (JV0812) [inline blog-service/2023-05-25-metrics.md:26]: Change the Y-Axis Source to **Dimension**.

File: blog-service/2023-05-25-metrics.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+We've enhanced the Heatmap functionality with the addition of custom dimensions on the Y-Axis. This feature enables you to gain deeper insights into your data by visualizing the frequency of occurrences for specific categories within a selected time range.
+
+Using the **Dimensional Heatmap**, you can now not only see the number of errors that occurred within a chosen time range (as with the regular Heatmap) but also effortlessly identify the specific categories, such as cluster/cpu/collector/sourceCategory, where these errors occurred most frequently.
+
+### Configure Dimension Heatmap
+
+To enable the Dimensional Heatmap with Categories on the Y-Axis, follow these steps:
+1. Run a metrics query or logs query with the **transpose** operator.
```
Comment (JV0812) [inline blog-service/2023-05-25-metrics.md:24]: Can you increase the image width to 800.

File: blog-service/2023-05-25-metrics.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+<img src={useBaseUrl('img/metrics/custom-heat-map.png')} alt="icon" width="540"/>
+2. Change the Y-Axis Source to **Dimension**
+3. A new field, **Dimension** will appear. Select the desired dimension from the list to be used as the source for the Y-Axis.
+<img src={useBaseUrl('img/metrics/dimension.png')} alt="icon" width="340"/>
```
Comment (JV0812) [inline blog-service/2023-05-25-metrics.md:28]: Can you reduce the image width to 300.

File: blog-service/2023-05-25-metrics.md (hunk 1 above)
Comment (JainM6) [reply blog-service/2023-05-25-metrics.md:26]: What's the change you are suggesting?

File: blog-service/2023-05-25-metrics.md (hunk 3 above)
Comment (JainM6) [reply blog-service/2023-05-25-metrics.md:28]: This is okay. I'm going with this.

File: blog-service/2023-05-25-metrics.md (hunk 1 above)
Comment (JV0812) [reply blog-service/2023-05-25-metrics.md:26]: Period is missing at the end of step 2.

Comment (JainM6) [reply blog-service/2023-05-25-metrics.md:26]:
```suggestion
2. Change the Y-Axis Source to **Dimension**.
```

Comment (JainM6) [reply blog-service/2023-05-25-metrics.md:26]: Commit suggestion
That's a helpful suggestion. Please use the Add Suggestion button for these kinds of suggestions. Please see the gif I've attached here.

Comment (JV0812) [reply blog-service/2023-05-25-metrics.md]: Sure, I will follow this going forward. I was lost finding that button, Thanks for this.

File: blog-service/2023-05-25-metrics.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+Using the **Dimensional Heatmap**, you can now not only see the number of errors that occurred within a chosen time range (as with the regular Heatmap) but also effortlessly identify the specific categories, such as cluster/cpu/collector/sourceCategory, where these errors occurred most frequently.
+
+### Configure Dimension Heatmap
+
+To enable the Dimensional Heatmap with Categories on the Y-Axis, follow these steps:
+1. Run a metrics query or logs query with the **transpose** operator.
+<img src={useBaseUrl('img/metrics/custom-heat-map.png')} alt="icon" width="540"/>
+2. Change the Y-Axis Source to **Dimension**.
+3. A new field, **Dimension** will appear. Select the desired dimension from the list to be used as the source for the Y-Axis.
+<img src={useBaseUrl('img/metrics/dimension.png')} alt="icon" width="340"/>
```
Comment (kimsauce) [inline blog-service/2023-05-25-metrics.md:28]:
```suggestion
3. A new field, **Dimension** will appear. Select the desired dimension from the list to be used as the source for the Y-Axis.<br/><img src={useBaseUrl('img/metrics/dimension.png')} alt="icon" width="340"/>
```

File: blog-service/2023-05-25-metrics.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+We've enhanced the Heatmap functionality with the addition of custom dimensions on the Y-Axis. This feature enables you to gain deeper insights into your data by visualizing the frequency of occurrences for specific categories within a selected time range.
+
+Using the **Dimensional Heatmap**, you can now not only see the number of errors that occurred within a chosen time range (as with the regular Heatmap) but also effortlessly identify the specific categories, such as cluster/cpu/collector/sourceCategory, where these errors occurred most frequently.
+
+### Configure Dimension Heatmap
+
+To enable the Dimensional Heatmap with Categories on the Y-Axis, follow these steps:
+1. Run a metrics query or logs query with the **transpose** operator.
+<img src={useBaseUrl('img/metrics/custom-heat-map.png')} alt="icon" width="540"/>
```
Comment (kimsauce) [inline blog-service/2023-05-25-metrics.md:25]:
```suggestion
1. Run a metrics query or logs query with the **transpose** operator.<br/><img src={useBaseUrl('img/metrics/custom-heat-map.png')} alt="icon" width="540"/>
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
minor fixes

**Review by kimsauce — APPROVED** [review]

**Pull request merged**